lbrynet/daemon/Daemon.py

```diff
@@ -1435,18 +1471,21 @@ def jsonrpc_resolve(self, force=False, uri=None, uris=[]):
 
     @AuthJSONRPCServer.auth_required
     @defer.inlineCallbacks
-    def jsonrpc_get(self, uri, file_name=None, timeout=None):
+    def jsonrpc_get(self, uri, file_name=None, timeout=None, delete_on_timeout=True):
         """
         Download stream from a LBRY name.
 
         Usage:
             get <uri> [<file_name> | --file_name=<file_name>] [<timeout> | --timeout=<timeout>]
+                      [<delete_on_timeout> | --delete_on_timeout=<delete_on_timeout>]
```

Comment:
whats the point of this flag? does anyone actually want to do this? cant they just use `file_delete`?

Reply:
Agreed, this seems backwards. We're supposed to be moving towards an asynchronous get (#854), not adding more synchronicity.